udata/models/md_fields.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from mongoengine.fields import StringField
+from udata.frontend.markdown import mdstrip
+
+class MDStripField(StringField):
```

Comment:
As stripping/truncation behavior is optional, let's call it `MarkdownField` which is more explicit ;)